db/fixtures/chargeback_rates.yml

```diff
@@ -17,6 +17,18 @@
           :finish: Infinity
           :fixed_rate: 0.0
           :variable_rate: 0.02
+    - :description: Used CPU Cores
+      :group: cpu
+      :source: used
+      :metric: cpu_usage_rate_average
```

Comment:
@zeari, I think you have not understood me. It is not a matter of being creating metrics. Each metric corresponds to a value of C&U stored in metrics_rollups. Also each metric is a hash columns in the Chargeback class. With this code you have just added a metric in the chargeback but will not have any functionality in the chargeback report. The code about this feature in the report is what I would look.